backend/api/Models/Parcel/ParcelModel.cs

```diff
@@ -9,7 +9,7 @@ public class ParcelModel : BaseModel
 
         public string PID { get; set; }
 
-        public string PIN { get; set; }
+        public int? PIN { get; set; }
```

Comment:
Updated to match the database column type.  Might need to change database to a string as I don't know the format used by the client.